src/software/ai/hl/stp/tactic/pivot_kick/pivot_kick_fsm.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include "software/ai/hl/stp/tactic/dribble/dribble_fsm.h"
+#include "software/ai/hl/stp/tactic/move/move_fsm.h"
+#include "software/ai/hl/stp/tactic/tactic.h"
+#include "software/logger/logger.h"
+
+struct PivotKickFSM
```

Comment:
What's the purpose of the PivotKickFSM? Why can't we reuse KickFSM and ChipFSM?

Reply:
There's nothing wrong with KickFSM/ChipFSM but PivotKickFSM seems like a better way to kick since it doesn't try to lose possession of the ball before kicking